core/tests/webdriverio_desktop/explorationHistoryTab.js

```diff
@@ -725,6 +725,7 @@ describe('Exploration history', function() {
     // Revert to version 2.
     await explorationEditorPage.navigateToHistoryTab();
     await explorationEditorHistoryTab.revertToVersion(2);
+    await explorationEditorHistoryTab.expectRevertToVersion(2);
```

Comment:
We need to wait till the exploration is reverted else we are not able to move to player because revert is in process. If we will look at the screenshot then we are still at history tab but we should be present on player in order to get the conversation.
![Screenshot from 2022-10-31 12-55-12](https://user-images.githubusercontent.com/86483059/198953897-6c12841b-465d-478e-b65b-357c5d3d6f62.png)

Now we are moving to player tab only after revert is done which means the message is present on the screen (reverted to version 2). This was able to resolve the flake.